Source/stores.cpp

```diff
@@ -44,6 +44,9 @@ ItemStruct smithitem[SMITH_ITEMS];
 int stextdown;
 char stextscrlubtn;
 char stextflag;
+ItemStruct smithbuybackitems[SMITH_ITEMS];
+ItemStruct witchbuybackitems[SMITH_ITEMS];

```

Comment:
Forgive me as I don't know this code base at all, shouldn't this be "WITCH_ITEMS"

Reply:
yup

Reply:
I use the same function for smith buy back and witch buy back, so I user the same value for this. It's both 25 at the moment. It looks a but ugly though. Alternatively I could just hardcode it or introduce a new def

Reply:
I'll have a look at it again - I did that during bug chasing, it shouldn't be necessary anymore.

Reply:
should definatly be  changed one way or the other :)

Reply:
> should definatly be  changed one way or the other :)

Done already :)